robot-tests/authentication/signUpAndSignIn.robot

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+#
+# SPDX-License-Identifier: MIT
+
+*** Settings ***
+
+Resource    ./keywords.robot
+
+* Test Cases *
+
+Test SignUp on localhost
+    Skip
+    Open Aalto Grades on Localhost
+    Navigate to SignUp
+    Fill Admin Register Information
+    Verify Admin User registering succeeded
+    Sign Out
+
+Test Login on localhost
+    Skip
```

Comment:
There is no need to skip these tests, as they are functional

Reply:
Test Login on localhost and Test SignUp on localhost

Reply:
My mistake, I thought they failed in GitHub Actions but I had skipped the sign up test so the login failed by default.